datacenter/ucp/2.1/guides/release-notes/index.md

```diff
@@ -12,6 +12,129 @@ known issues for the latest UCP version.
 You can then use [the upgrade instructions](../admin/upgrade.md), to
 upgrade your installation to the latest release.
 
+## Version 2.1.7
+
+(12 February 2018)
+
+**Security Notice**
+
+The log-driver is now disabled for containers started by backup operations.
```

Comment:
this is not accurate.  We are forcing the customer to use `--log-driver=none` when doing backup.